lib/cartopy/tests/mpl/test_images.py

```diff
@@ -150,6 +152,21 @@ def test_imshow_wrapping():
     assert ax.get_xlim() == (-180, 180)
 
 
+def test_imshow_rgba():
+    # tests that the alpha of a RGBA array passed to imshow is set to 0
+    # instead of masked
+    z = np.random.rand(100, 100)
+    cmap = cm.get_cmap()
+    norm = colors.Normalize(vmin=0, vmax=1)
+    z1 = cmap(norm(z))
+    plt_crs = ccrs.LambertAzimuthalEqualArea()
+    latlon_crs = ccrs.PlateCarree()
+    ax = plt.axes(projection=plt_crs)
+    ax.set_extent([-30, -20, 60, 70], crs=latlon_crs)
+    img = ax.imshow(z1, extent=[-26, -24, 64, 66], transform=latlon_crs)
+    assert sum(img.get_array().data[:, 0, 3]) == 0
```

Comment:
Maybe we should also test the RGB (n, m, 3) case in addition to the RGBA (n, m, 4) cmap generated case?

Reply:
OK